### DISCUSSION TITLE
Usage for anomaly detection in images

### DISCUSSION THREAD

**Discussion by alhussein-jamil** [yzhao062/pyod#578] [General]
Hello,

I find this repository quite interesting. However, I'm having trouble identifying a standard method for detecting anomalies in images. Specifically, I have a dataset containing both normal and abnormal images. Could you advise on the recommended pipeline for performing binary classification using the pyod framework? My goal is to implement this for industrial anomaly detection.

Thanks!

**Comment by haimat** [comment]
Heyho, we have the same requirement.
Have you ever figured out how to apply pyod on image datasets?
